samples/ComponentExamples/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../common/config/tsc/tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist/dist-esm",
+    "baseUrl": "./src",
+    "paths": {
+      "@azure/communication-react": ["../../../packages/communication-react/src"]
```

Comment:
neat!

But might be worth addinng more paths for the packlets? This still means that we have to `rush build -t communication-react` for hot reloading to work.

Reply:
See tsconfig.json for samples/Chat.

Reply:
I also took a small experiment locally, looks like hot load is working fine - I guess it is handled by packages\communication-react\tsconfig.json - maybe we should remove those paths in chat as well